### ISSUE TITLE
https://metacpan.org/release/Compress-LZ4 is missing

### ISSUE THREAD

**Issue by chorny** [metacpan/metacpan-web#1827]
It is shown in the search: https://metacpan.org/search?q=Compress-LZ4

02packages.details.txt:
```
Compress::LZ4                      0.23  G/GR/GRAY/Compress-LZ4-0.23.tar.gz
```
Compress::LZ4 is short, but not pod-only.

**Comment by haarg** [comment]
I've addressed part of this in #1828, where the page https://metacpan.org/pod/Compress::LZ4 was including a non-working link to https://metacpan.org/release/Compress-LZ4.

**Comment by haarg** [comment]
I've reindexed the dist to fix this.
